lib/models/statement.js

```diff
@@ -187,6 +188,10 @@ schema.statics.aggregateByAuth = function aggregateByAuth(
     if (authInfo.client && authInfo.client.lrs_id) {
       parsedPipeline.unshift({ $match: { lrs_id: objectId(authInfo.client.lrs_id) } });
     }
+
+    if (sampleSize) {
```

Comment:
```suggestion
    if (sampleSize !== undefined && sampleSize > 0) {
```

Reply:
@davidavdav this has been through QA, can you please just commit this suggestion so that this condition is a little clearer? Then we're happy to merge and release this 👍

Reply:
Strange... in the email from github the suggestion is 
```
if (sampleSize !== undefined && sampleSize > 0)
```
whereas in the github PR-page the suggestions has been reduced to 
```
if (sampleSize > 0)
```
I suppose `undefined > 0 === false`, anyway, so I'll click on the "Commit suggestion" button and see how that works...

Reply:
Yeah sorry I changed the suggestion after realising `undefined > 0` would be `false`.

Reply:
Thanks for committing so quickly. I'll wait for the build to pass then merge.